lib/initializers/errors.js

```diff
@@ -32,11 +39,7 @@ Initializer.add('startup', 'stex.request-domains', ['stex.express', 'stex.loggin
   stex.app.use(function (req, res, next) {
     var requestDomain = domain.create();
     var reportErrorDirectly = function(err) {
-        log.error(err.message);
-        // report to sentry if we are enabled
-        if(sentry) {
-            sentry.captureError(err);
-        }
+        stex.reportError;
```

Comment:
I don't think you tested this :)

`stex.reportError(err)`, not `stex.reportError`